tsconfig.json

```diff
@@ -1,18 +1,12 @@
 {
-  "compilerOptions": {
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
-    "noUnusedLocals": true,
-
-    "lib": ["es2017", "dom"],
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "target": "es2017",
-    "outDir": "dist",
-    "sourceMap": true,
-    "declaration": true
-  },
-  "include": ["src", "test"]
+  "extends": "./node_modules/@loopback/build/config/tsconfig.common.json",
+  "include": [
+    "src",
+    "test"
+  ],
+  "exclude": [
+    "node_modules/**",
+    "packages/*/node_modules/**",
+    "**/*.d.ts"
```

Comment:
Isn't this the default configuration of `tsc`? Why is it necessary to list these exclude items now, when the build was working well without them before?